corehq/apps/registration/templates/registration/partials/register_new_user/submit_info.html

```diff
@@ -10,12 +10,12 @@ <h2>{% trans "Creating Account..." %}</h2>
       Please wait a moment&mdash;almost there!
     {% endblocktrans %}
   </p>
+  <p class="timeout-message" data-bind="visible: showSecondTimeout">
```

Comment:
this is basically filler test. I removed Enabled Web Apps Trial because the trial will no longer exist